src/components/DiaryCard.tsx

```diff
@@ -0,0 +1,50 @@
+import {Button,Typography,CardContent,CardActions, Card} from '@mui/material';
+import  React  from "react";
+import { useState } from 'react';
+
+const DiaryCard = (props:any) => {
```

Comment:
Do not use any. Define a data type by an interface

Reply:
updated